src/api/Controllers/BoreholeFileController.cs

```diff
@@ -89,6 +90,69 @@ public async Task<IActionResult> Download([Range(1, int.MaxValue)] int boreholeF
         }
     }
 
+    /// <summary>
+    /// Get the borehole file information for the data extraction file with the provided boreholeFileId and page index.
+    /// </summary>
+    /// <param name="boreholeFileId">The id of the borehole file.</param>
+    /// <param name="index">The index of the page in the borehole file, with 1 as index for the first page.</param>
+    /// <returns>The name and size of the selected image as well as the total image count for the borehole file.</returns>
+    [HttpGet("getDataExtractionFileInfo")]
+    [Authorize(Policy = PolicyNames.Viewer)]
+    public async Task<IActionResult> GetDataExtractionFileInfo([Required, Range(1, int.MaxValue)] int boreholeFileId, int index)
+    {
+        if (boreholeFileId == 0) return BadRequest("No boreholeFileId provided.");
```

Comment:
Q: wofür brauchts diesen Fall, wann kann das vorkommen? Wird es nicht übers not found abgedeckt?

Reply:
Sollte das nicht übers "required" schon abgedeckt sein? Und in dem Fall genau den BadRequest mit einer ähnliche lautenden Fehlermeldung zurückgeben?

Beim Test schreiben kannst du das ja dann mal testen ;)